brainstormforce/astra
# AST-2792 - fix: Support for button preset outline (buttons 4-6) styles in editor

### Description

- The problem occurs with button presets outline styles, editor shows bg color even for outline design preset (buttons 4-5), which is not expected.
- JIRA - https://brainstormforce.atlassian.net/browse/AST-2792

### Screenshots

https://d.pr/v/exFxcf
### Types of changes


- bugfix - The problem occurs with button presets outline styles, editor shows bg color even for outline, which is not expected.



### How has this been tested?

- Global > Buttons > Button Presets
- Select a outline design from available button design options.
- Check in frontend & compare with editor.
- The buttons should look same in terms of design

### Checklist:
